template.distillery/README.md

```diff
@@ -222,3 +228,24 @@ you update the server:
 ```
 make APP_SERVER=http://${YOUR_SERVER} APP_REMOTE={yes|no} chcp
 ```
+
+## Use Makefile.local file.
+
+You need to define `APP_REMOTE` and `APP_SERVER` each time you want to build
+mobile applications or to update it.
+
+If you don't want to pass every time the variables `APP_SERVER` and
+`APP_REMOTE`, you can change the values of these variables in
+`Makefile.local.example` and rename this file in `Makefile.local`. With this
+method, `APP_REMOTE` and `APP_SERVER` variables are not mandatory when you build
+or update the mobile application. You can use:
+```
+make chcp
+make run-android
+make run-ios
+...
+```
+
+It is recommended to use this file to add rules or variables only used for
+local development and it must not be deployed or shared (by default, this file
```

Comment:
I would rephrase this, _e.g._, as follows: _this file is meant for rules and variables that are only relevant for local development..._